src/plugins/wb-jsonmanager/jsonmanager.js

```diff
@@ -210,12 +210,15 @@ var componentName = "wb-jsonmanager",
 			}
 		],
 		opsRoot: [],
-		settings: { }
+		settings: { },
+		jsonSource: { },
```

Comment:
No need for a global because this variable is only use inside `extractData` function.